thirdparty/misc/open-simplex-noise.c

```diff
@@ -100,27 +100,27 @@ static const signed char gradients4D[] = {
 	-3, -1, -1, -1,     -1, -3, -1, -1,     -1, -1, -3, -1,     -1, -1, -1, -3,
 };
 
-static double extrapolate2(struct osn_context *ctx, int xsb, int ysb, double dx, double dy)
+static double extrapolate2(const struct osn_context *ctx, int xsb, int ysb, double dx, double dy)
 {
-	int16_t *perm = ctx->perm;	
+        const int16_t *perm = ctx->perm;
```

Comment:
That's not the same indentation as upstream: https://github.com/smcameron/open-simplex-noise-in-c/commit/826f1dd1724e6fb3ff45f58e48c0fbae864c3403